caom2pipe/tests/data/config.yml

```diff
@@ -1,5 +1,4 @@
 working_directory: /usr/src/app/caom2pipe/caom2pipe/tests/data
-netrc_filename: test_netrc
```

Comment:
No. It should be gone from all the places in the code where it means "archive", but it might still be called "archive", and actually mean "collection". That's another thing I'm trying to clean up as I go.

Do you think I should just go through the code and do that renaming on this commit?